Client/Frontend/Browser/Browser.swift

```diff
@@ -47,4 +58,5 @@ class Browser {
     func loadRequest(request: NSURLRequest) {
         webView.loadRequest(request)
     }
+    
```

Comment:
Please remove newline (we generally do not add extra lines before or after curly braces).